### PR TITLE
Added pretty printed tab for graphQL

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added pretty printed tab for graphQL

I dropped the 'request' on tabs with GraphQL so they would fit across the page.

### CODE REVIEW

File: docs/Reference/Pantheon-API-Methods.md [hunk 1]
```diff
@@ -1538,10 +1704,37 @@ Returns information about the block by hash.
...
+        transactions {
+          hash
+        }
+        timestamp
+        difficulty
+        totalDifficulty
+        gasUsed
+        gasLimit
+        hash
+        noce
```
Comment (NicolasMassart) [inline docs/Reference/Pantheon-API-Methods.md]:
```suggestion
        nonce
```

File: docs/Reference/Pantheon-API-Methods.md [hunk 2]
```diff
@@ -1538,10 +1704,37 @@ Returns information about the block by hash.
     }
     ```
     
-    ```bash tab="curl GraphQL request"
+    ```bash tab="curl GraphQL"
     curl -X POST -H "Content-Type: application/json" --data '{ "query": "{block (hash : \"0xb0efed1fc9326fee967cb2d845d4ebe57c5350a0670c8e86f8052dea6f219f92\") {number transactions{hash} timestamp difficulty totalDifficulty gasUsed gasLimit hash noce ommerCount logsBloom mixHash ommerHash extraData stateRoot receiptsRoot transactionCount transactionsRoot}}"}' http://localhost:8547/graphql
```
Comment (NicolasMassart) [inline docs/Reference/Pantheon-API-Methods.md]:
```suggestion
    curl -X POST -H "Content-Type: application/json" --data '{ "query": "{block (hash : \"0xb0efed1fc9326fee967cb2d845d4ebe57c5350a0670c8e86f8052dea6f219f92\") {number transactions{hash} timestamp difficulty totalDifficulty gasUsed gasLimit hash nonce ommerCount logsBloom mixHash ommerHash extraData stateRoot receiptsRoot transactionCount transactionsRoot}}"}' http://localhost:8547/graphql
```

File: docs/Reference/Pantheon-API-Methods.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Reference/Pantheon-API-Methods.md]: Fixed

File: docs/Reference/Pantheon-API-Methods.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/Pantheon-API-Methods.md]: Fixed

**Pull request merged**